orangecontrib/prototypes/widgets/owfeaturestatistics.py

```diff
@@ -0,0 +1,512 @@
+            if continuous_f and x[:, cont_var_idx].size:
+                result[cont_var_idx] = continuous_f(x[:, cont_var_idx])
+            if time_f and x[:, time_var_idx].size:
+                result[time_var_idx] = time_f(x[:, time_var_idx])
+            if string_f and x[:, str_var_idx].size:
+                result[str_var_idx] = string_f(x[:, str_var_idx])
+            return result
+
+        # Compute the first moment
+        _first_moment = partial(
+            _apply_to_types,
+            discrete_f=lambda x: ss.mode(x)[0],
+            continuous_f=lambda x: ut.nanmean(x, axis=0),
+        )
+        self._first_moment = np.hstack(map(_first_moment, matrices))
+
+        # Compute the second moment
+        _second_moment = partial(
+            _apply_to_types,
+            discrete_f=ss.entropy,
+            continuous_f=lambda x: ut.var(x, axis=0),
+        )
+        self._second_moment = np.hstack(map(_second_moment, matrices))
+
+        # Compute # of missing values
+        _missing = partial(
+            _apply_to_types,
+            discrete_f=lambda x: ut.countnans(x, axis=0),
+            continuous_f=lambda x: ut.countnans(x, axis=0),
+            string_f=lambda x: (x == StringVariable.Unknown).sum(axis=0),
+        )
+        self._missing = np.hstack(map(_missing, matrices))
+
+    def headerData(self, section, orientation, role):
+        # type: (int, Qt.Orientation, Qt.ItemDataRole) -> Any
+        if orientation == Qt.Horizontal:
+            if role == Qt.DisplayRole:
+                return self.Columns.from_index(section).name
+
+    def data(self, index, role):
+        # type: (QModelIndex, Qt.ItemDataRole) -> Any
+        if not index.isValid():
+            return
+
+        row, column = index.row(), self.Columns.from_index(index.column())
+        # Make sure we're not out of range
+        if not 0 <= row <= self._n_attributes:
+            return QVariant()
+
+        output = None
+        attribute = self._attributes[row]
+
+        if column == self.Columns.ICON:
+            if role == Qt.DecorationRole:
+                return gui.attributeIconDict[attribute]
+        elif column == self.Columns.NAME:
+            if role == Qt.DisplayRole:
+                output = attribute.name
+        elif column == self.Columns.DISTRIBUTION:
+            if role == self.DistributionRole:
+                if isinstance(attribute, (DiscreteVariable, ContinuousVariable)):
+                    return attribute, self._data
+        elif column == self.Columns.FIRST_MOMENT:
+            if role == Qt.DisplayRole:
+                output = self._first_moment[row]
+        elif column == self.Columns.SECOND_MOMENT:
+            if role == Qt.DisplayRole:
+                output = self._second_moment[row]
+        elif column == self.Columns.MISSING:
+            if role == Qt.DisplayRole:
+                output = '%d / %d (%d%%)' % (
+                    self._missing[row], self._n_instances,
```

Comment:
`self._n_instances` is the same for all features? Then better move the number of instances into the control area info box and show here only the absolute count and percent value.